coss/home/static/css/coss.scss

```diff
@@ -41,6 +41,14 @@ h1, h2, h3, h4 {
     }
 }
 
+.container {
+
+    @media (min-width: 1500px) {
```

Comment:
Shouldn't this be something smaller like 1441px?

Reply:
It's [usual](https://getbootstrap.com/docs/4.0/layout/grid/#grid-options) for the container width to be a bit smaller than the breakpoint to account for existing margins too.